pkg/asset/machines/master.go

```diff
@@ -159,7 +160,14 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 		}
 
 		mpool := defaultAWSMachinePoolPlatform()
-		mpool.AMIID = string(*rhcosImage)
+
+		osImage := strings.SplitN(string(*rhcosImage), ",", 2)
```

Comment:
Pulling the AMIID out of the `rhcosImage` string seems to happen in three different locations.  Does it make sense for this to be a function?